src/client.ts

```diff
@@ -222,6 +243,19 @@ export class Client {
     return this.executeRequestOperation(operation);
   };
 
+  mutation({
+    query,
```

Comment:
I'd just expand these into three arguments, the reason being that they're likely to be three separate variables in user code anyway.